### DISCUSSION TITLE
Display active battery profile on the OSD

### DISCUSSION THREAD

**Discussion by helicocrasher** [iNavFlight/inav#7840] [Q&A] [answered]
Can I show/see the active Battery Profile in the OSD?

**Comment by MrD-RC** [comment, marked as answer]
No, there is no way to do this currently.

**Comment by helicocrasher** [comment]
I tempted to ask/propose Batttery Profile Display in OSD  :-)

**Reply by MrD-RC** [reply]
Go for it. It would be easy enough to add, just using the batter symbol followed by the profile number. Just raise the issue 🙂
